### ISSUE TITLE
Homebridge Hue plugin update needs Node.js v16.17.0

### ISSUE THREAD

**Issue by jaxter0ne** [oznu/homebridge-syno-spk#102]
Hi,

I'm trying to update the Homebridge Hue plugin but it says it recommends Node.js v16.17.0 and that the current installed version is v16.16.0.
I can see that I'm on the latest update of the package. Is there a new update coming soon? Can I update Node.js manually? Or can I update the plugin without updating Node.js?

Thank you!

**Comment by datom844** [comment]
Hi, 

in Homebridge go to the Menu with the 3 dots. Start Console and type "hb-service update-node"

**Comment by jaxter0ne** [comment]
Hi,

So I tried that and my NodeJS version number actually decreased for some reason! haha and then when I tried doing it again it said I couldn't do it and that I had to pull the latest docker image. So I did that : https://www.wundertech.net/how-to-update-docker-containers-on-a-synology-nas/

And it worked! In case someone runs into the same problem...

> Hi,
> 
> in Homebridge go to the Menu with the 3 dots. Start Console and type "hb-service update-node"
